etc/kayobe/ansible/requirements.yml

```diff
@@ -4,7 +4,8 @@ collections:
     version: 0.3.0
   - name: pulp.squeezer
     version: 0.0.11
+  - name: stackhpc.cephadm
+    version: 1.8.0
 roles:
   - name: vxlan
     src: https://github.com/stackhpc/ansible-role-vxlan.git
```

Comment:
If you want to use the release on galaxy, it should look like this:

```
- src: stackhpc.vxlan
```

The playbook would need to switch to `role: stackhpc.vxlan`